test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-1.xml

```diff
@@ -36,11 +36,13 @@
 	<variable name="CACHE_DIR" value="cacheDir=."/>
 	
 	<variable name="currentMode" value="$mode204$"/>
+	<variable name="lambdaMode" value="-Xtrace:print={j9shr.2259}"/>
```

Comment:
No need to add lambdaMode here. You can directly add -Xtrace:print={j9shr.2259} to the commands